changelogs/CHANGELOG.rst

```diff
@@ -11,6 +11,8 @@ v2.0.0
 Major Changes
 -------------
 
+- Please refer to ansible.netcommon `changelog <https://github.com/ansible-collections/ansible.netcommon/blob/main/changelogs/CHANGELOG.rst#ansible-netcommon-collection-release-notes>`_ for more details.
+- Requires ansible.netcommon v2.0.0+ to support `ansible_network_single_user_mode` and `ansible_network_import_modules`.
```

Comment:
I don't know why the changelog tool swaps the order for this fragment. But reversing it probably makes more sense for the user to understand why this is a major release.

Reply:
okay

Reply:
Ack